### PR TITLE
Added map plot and crime count

### LINKED ISSUE #18: Vancouver map chart coding

### PR DESCRIPTION
**Pull request by MoNorouzi23**
I added map plot as well as total crime count and top 3 of it!please review and merge it. I removed Sharon's part due to avoiding merge conflicts!

### CODE REVIEW

**Review by WaleedMahmood1 — APPROVED** [review]
Thanks for updating the app.py with the map code @MoNorouzi23! I have made some slight additions back into the `app.py` file - of the imports, data reading and processing code - to ensure it deploys in the PR so that we can view the deployed dashboard preview before we merge into main.

File: src/app.py [hunk 1]
```diff
@@ -70,11 +110,41 @@
...
 
+@app.callback(
+    Output('crime-map-chart', 'figure'),
+    [Input('crime-type-dropdown', 'value'),
+     Input('neighbourhood-dropdown','value')]
+)
+
+def update_map_chart(selected_crime,selected_neighbourhood):
+
+    if selected_neighbourhood == 'All':
```
Comment (786213750) [inline src/app.py:125]: you may able to simplify this, ref #31

File: src/app.py [hunk 2]
```diff
@@ -4,56 +4,96 @@
...
+        html.H1('VANCOUVER CRIME TRACKER 2023', style={'color': 'purple'}),
         html.Br(),
     ]),
     dbc.Row([
         dbc.Col([
-            html.H1('Total Crime Count'),
-        html.Br(),
+            dbc.Card([
+                dbc.CardBody([
+                    html.H3('Total Crime Count', style={'color': 'blue'}),
```
Comment (786213750) [inline src/app.py:30]: what is the color we should stick to?

**Review by 786213750 — APPROVED** [review]
Nicely done!

File: src/app.py (hunk 2 above)
Comment (MoNorouzi23) [reply src/app.py:30]: I tried different colours but we can think about it!

**Pull request merged**